content/builds/hunter/liars-handshake.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Liars Handshake Hunter-Titan"
+description: "PvE build with Liar's Handshake"
+season: 15
+draft: false
+tags: ['hunter', 'pve', "liar's handshake"]
+d2_armor_picker_hash: 'n/a'
```

Comment:
almost forgot! just leave this as an empty string or remove it.  the value will automatically add a section that will link to d2armorpicker with the hash info, but the hash won't do anything here.